gensrc/proto/lake_types.proto

```diff
@@ -116,11 +120,17 @@ message TxnLogPB {
         optional DelvecMetadataPB delvec_meta = 4;
     }
 
+    message OpAlterMeta {
+        required TabletMetaTypePB table_meta_type = 1;
```

Comment:
Required protobuf fields are prohibited in StarRocks, please use optional instead.